src/common/source.rs

```diff
@@ -15,7 +15,7 @@ use std::{
 #[derive(Debug, PartialEq, Eq)]
 pub struct Source {
     pub contents: String,
-    pub path:     PathBuf,
```

Comment:
Ditto `struct` fields, `match` arms, and `let` expressions. What can I say, I like it when things align nicely ;P